infrastructure/smart-contracts/scripts/upgradeNodeLicense.mjs

```diff
@@ -1,15 +1,26 @@
 import hardhat from "hardhat";
 const { ethers, upgrades } = hardhat;
 const address = "0xbc14d8563b248B79689ECbc43bBa53290e0b6b66";
+const xaiAddress = "0x4Cb9a7AE498CEDcBb5EAe9f25736aE7d428C9D66";
+const esXaiAddress = "0x4C749d097832DE2FEcc989ce18fDc5f1BD76700c";
+const ethChainLinkFeedAddress = "0x639Fe6ab55C921f74e7fac1ee960C0B6293ba612";
+const xaiChainLinkFeedAddress = "0x806c532D543352e7C344ba6C7F3F00Bfbd309Af1";
```

Comment:
I know we have the address in a lot of scripts, but for adding updates to these scripts we should start adding the import of the config.ts just to be 100% sure we don't mix up contract addresses here.

Just for the next time concider adding the config.ts import and going from there. I think it has this in the `deployStakingV2.mjs`

Reply:
This has been updates to use the config files.